test/cypress/integration/web-payments/google-pay.test.cy.js

```diff
@@ -369,4 +371,35 @@ describe('Google Pay payment flow', () => {
       })
     })
   })
+
+  describe('Google Pay Welsh language payment flow', () => {
+    describe('Secure card payment page', () => {
+      it('Should show Google Pay as a payment option in Welsh', () => {
+        cy.task('setupStubs', [...chargeStubsWithGooglePayOrApplePayEnabled(false, false)])
+        cy.visit(`/secure/${tokenId}`)
+
+        cy.location('pathname').should('eq', `/card_details/${chargeId}`)
+
+        cy.log('Should show Google Pay as a payment option and user chooses it')
+
+        cy.task('clearStubs')
+        cy.task('setupStubs', [...chargeStubsWithGooglePayOrApplePayEnabled(true, false, null, null, 'cy'), worldpay3dsFlexDdcStub])
+
+        cy.visit(`/card_details/${chargeId}`, {
+          onBeforeLoad: win => {
+            if (win.PaymentRequest) {
+              cy.stub(win, 'PaymentRequest', getMockPaymentRequest(validPaymentRequestResponse))
+            } else {
+              win.PaymentRequest = getMockPaymentRequest(validPaymentRequestResponse)
+            }
+          }
+        })
+
+        cy.get('#wallet-options>.web-payment-button-section>.govuk-heading-l').should('contain', 'Rhowch fanylion taliad')
+        cy.get('#wallet-options>.google-pay-container>.govuk-heading-m').should('contain', 'Talu gyda Google Pay')
+        cy.get('#google-pay-payment-method-divider-word').should('contain', 'neu')
```

Comment:
Again, do we want an English equivalent in another Google Pay test?

Reply:
Added